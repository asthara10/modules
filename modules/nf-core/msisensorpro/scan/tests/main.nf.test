nextflow_process {

    name "Test Process MSISENSORPRO_SCAN"
    script "../main.nf"
    process "MSISENSORPRO_SCAN"

    tag "modules"
    tag "modules_nfcore"
    tag "msisensorpro"
    tag "msisensorpro/scan"

    test("test_msisensorpro_scan") {


        when {
            process {
                """
                
                input[0] = [
        [ id:'test', single_end:false ], // meta map
        file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
    ]

                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(process.out.versions).match("versions") }
                { assert snapshot(process.out).match() }
            )
        }

    }

}
