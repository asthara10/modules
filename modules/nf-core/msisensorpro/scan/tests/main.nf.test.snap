{
    "versions": {
        "content": [
            [
                "versions.yml:md5,51909a57a65cb89c0e8b3000366bde86"
            ]
        ],
        "timestamp": "2023-11-24T14:32:11.246335"
    }
}